app/controllers/maps_controller.rb

```diff
@@ -1,10 +1,23 @@
+# frozen_string_literal: true
+
 class MapsController < ApplicationController
+  def show
+    locations = Location.where(map_id: params[:id])
```

Comment:
I suppose `id` can be multiple IDs? If yes, naming it `ids` is clearer

Reply:
I don't understand what you mean? This just shows one map which only has one `id`.

Reply:
Misread, my bad. :D

Reply:
Haha